src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -6200,3 +6193,96 @@ def test_node_public_ip_tags(self, resource_group, resource_group_location):
         self.cmd(cmd, checks=[
             self.is_empty(),
         ])
+
+    @live_only()
+    @AllowLargeResponse()
+    def test_get_trustedaccess_roles(self):
+        versions_cmd = 'aks trustedaccess role list -l eastus -o json'
+        roles = self.cmd(versions_cmd).get_output_in_json()
+        assert len(roles) > 0
+        role = roles[0]
+        assert len(role['rules']) > 0
+    
+    @live_only()
```

Comment:
No need for this tag, you've added the recording file for this test, it could be executed successfully in CI.